### DISCUSSION TITLE
Results Comparison With BPG RGB Image Format

### DISCUSSION THREAD

**Discussion by ankitlat** [tensorflow/compression#89] [Q&A] [answered]
Hi Johannes,

I was wondering if you can share the comparison results of method bmshj2018-hyperprior-msssim-8 with BPG RGB image format. Or is there any particular reason for not comparing methods with BPG when image is given as RGB format?

**Comment by jonaballe** [comment, marked as answer]
Hi @ankitlat, I think [this file](https://github.com/tensorflow/compression/blob/master/results/image_compression/tecnick/PSNR_sRGB_RGB/balle-2018-iclr-opt-msssim.txt) should have the numbers you want? I'm not entirely sure what you are asking for.
